phpunit.xml

```diff
@@ -42,22 +42,25 @@
   </php>
   <testsuites>
     <testsuite name="kernel">
-      <directory>./profiles/mukurtu-cms/modules/mukurtu_protocol/tests/src/Kernel</directory>
+      <directory>web/profiles/contrib/mukurtu-cms/modules/mukurtu_protocol/tests/src/Kernel</directory>
     </testsuite>
     <testsuite name="kernel">
-      <directory>./profiles/mukurtu-cms/modules/mukurtu_collection/tests/src/Kernel</directory>
+      <directory>web/profiles/contrib/mukurtu-cms/modules/mukurtu_collection/tests/src/Kernel</directory>
     </testsuite>
     <testsuite name="kernel">
-      <directory>./profiles/mukurtu-cms/modules/mukurtu_import/tests/src/Kernel</directory>
+      <directory>web/profiles/contrib/mukurtu-cms/modules/mukurtu_import/tests/src/Kernel</directory>
     </testsuite>
     <testsuite name="kernel">
-      <directory>./profiles/mukurtu-cms/modules/mukurtu_export/tests/src/Kernel</directory>
+      <directory>web/profiles/contrib/mukurtu-cms/modules/mukurtu_export/tests/src/Kernel</directory>
     </testsuite>
     <testsuite name="kernel">
-      <directory>./profiles/mukurtu-cms/modules/mukurtu_drafts/tests/src/Kernel</directory>
+      <directory>web/profiles/contrib/mukurtu-cms/modules/mukurtu_drafts/tests/src/Kernel</directory>
+    </testsuite>
+    <testsuite name="kernel">
+      <directory>web/profiles/contrib/mukurtu-cms/modules/original_date/tests/src/Kernel</directory>
     </testsuite>
```

Comment:
Looks like there was another set of tests in `original_date` module that we weren't running  yet on GitHub. They still seem to be passing, which is great.